### PR TITLE
Add Chapter for terms and definitions 

### PR DESCRIPTION
**Pull request by RincewindsHat**
#146 Add explanation for gABI and psABI

### CODE REVIEW

File: riscv-elf.md [hunk 1]
```diff
@@ -947,7 +947,7 @@ there is no differentiation between ILP32 and LP64 abis.
 
 # <a name=terms-definitions></a> Terms and Definitions
 
-* **ABI**: [Application binary interface](Application binary interface).
+* **ABI**: [Application binary interface](https://en.wikipedia.org/wiki/Application_binary_interface).
     Here a combination of the **gABI** and the **psABI**
 * **psABI**: processor specific ABI
 * **gABI**: general ABI
```
Comment (ebahapo) [inline riscv-elf.md:953]: s/general/generic/

File: riscv-elf.md [hunk 2]
```diff
@@ -947,7 +947,7 @@ there is no differentiation between ILP32 and LP64 abis.
 
 # <a name=terms-definitions></a> Terms and Definitions
 
-* **ABI**: [Application binary interface](Application binary interface).
+* **ABI**: [Application binary interface](https://en.wikipedia.org/wiki/Application_binary_interface).
     Here a combination of the **gABI** and the **psABI**
 * **psABI**: processor specific ABI
```
Comment (ebahapo) [inline riscv-elf.md:952]: Please, move this line after the one for the gABI.

Comment (RincewindsHat) [reply riscv-elf.md:953]: [x]

File: riscv-elf.md (hunk 1 above)
Comment (RincewindsHat) [reply riscv-elf.md]: [x] done in next patch

**Comment by ebahapo** [comment]
> This is OK with me. @ebahapo did you have further comments?

LGTM

**Pull request merged**